packages/@tailwindcss-postcss/src/index.test.ts

```diff
@@ -245,8 +245,8 @@ describe('@content', () => {
 
     expect(result.messages).toContainEqual({
       type: 'dir-dependency',
-      dir: expect.stringMatching(/example-project$/),
-      glob: '../other-project/src/**/*.js',
+      dir: expect.stringMatching(/other-project\/src$/),
+      glob: '**/*.js',
```

Comment:
We are optimizing the globs using the `get_fast_patterns` where we try to move as many static parts to the "base" of the glob, and only keep the dynamic parts in the glob.

In this case, `../other-project/src/` is attached to the base (and later resolved such that the `..` is gone). Then the `**/*.js` stays in the glob.